ftw/Dockerfile.ftw

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 The OWASP Coraza contributors
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ghcr.io/coreruleset/go-ftw:1.1.2
+
+RUN apk update && apk add curl jq
+
+WORKDIR /workspace
+
+ENV CRS_VERSION=4.5.0
+
+ADD https://github.com/coreruleset/coreruleset/archive/refs/tags/v${CRS_VERSION}.tar.gz /workspace/coreruleset/
+RUN cd coreruleset && tar -xf v${CRS_VERSION}.tar.gz --strip-components 1
+
+# to dynamically pull latest CRS version:
```

Comment:
Please remove the commented code. Dynamically using a different version would lead to a lot of pain when debugging it